### PR TITLE
[State] Update to alpha12 snapshot

### PR DESCRIPTION
**Pull request by florina-muntenescu**

### CODE REVIEW

File: StateCodelab/finished/src/main/java/com/codelabs/state/todo/TodoComponents.kt [hunk 1]
```diff
@@ -208,18 +209,22 @@ fun TodoInputText(
...
-    keyboardOptions = KeyboardOptions.Default.copy(imeAction = ImeAction.Done),
-    keyboardActions = KeyboardActions(onDone = {
-        onImeAction()
-//        softKeyboardController?.hideSoftwareKeyboard()
-    }),
-    modifier = modifier
-)
+) {
+    lateinit var softwareKeyboardController: SoftwareKeyboardController
+    return TextField(
```
Comment (chrisbanes) [inline StateCodelab/finished/src/main/java/com/codelabs/state/todo/TodoComponents.kt:214]: remove the `return`

File: StateCodelab/start/src/main/java/com/codelabs/state/todo/TodoComponents.kt [hunk 2]
```diff
@@ -208,18 +209,22 @@ fun TodoInputText(
...
-    keyboardOptions = KeyboardOptions.Default.copy(imeAction = ImeAction.Done),
-    keyboardActions = KeyboardActions(onDone = {
-        onImeAction()
-//        softKeyboardController?.hideSoftwareKeyboard()
-    }),
-    modifier = modifier
-)
+) {
+    lateinit var softwareKeyboardController: SoftwareKeyboardController
+    return TextField(
```
Comment (chrisbanes) [inline StateCodelab/start/src/main/java/com/codelabs/state/todo/TodoComponents.kt:214]: remove the `return`

**Review by chrisbanes — COMMENTED** [review]
Probably needs `spotlessApply` running

**Comment by florina-muntenescu** [comment]
> Probably needs `spotlessApply` running

We don't have spotless here. I formatted the code manually though

**Review by chrisbanes — APPROVED** [review]

**Pull request merged**